packages/EasyCodingStandard/src/Error/ErrorAndDiffCollector.php

```diff
@@ -55,7 +56,7 @@ public function addErrorMessage(SmartFileInfo $fileInfo, int $line, string $mess
     {
         $this->changedFilesDetector->invalidateFileInfo($fileInfo);
 
-        $relativePathnameToRoot = Strings::substring($fileInfo->getRealPath(), strlen(getcwd()) + 1);
+        $relativePathnameToRoot = rtrim((new Filesystem)->makePathRelative($fileInfo->getRealPath(), getcwd()), '/');
```

Comment:
What we need to do to use here `getRelativeFilePathFromDirectory()` method from `SmartFileInfo`?

To prevent duplications.